wpcom-thumbnail-editor.php

```diff
@@ -743,7 +743,7 @@ public function get_thumbnail_url( $existing_resize, $attachment_id, $size ) {
 				)
 			);
 		else
-			$url = wp_get_attachment_url( $attachment_id );
+			return $existing_resize;
```

Comment:
The `return` here means that for contexts where Jetpack isn't active, we never return the resized image. It's probably better to check for `defined( 'JETPACK_DEV_DEBUG' )` early in the function and return there.